VMware.CloudFoundation.CertificateManagement.psm1

```diff
@@ -25,17 +25,900 @@ if ($PSEdition -eq 'Desktop') {
 
     if ("TrustAllCertificatePolicy" -as [type]) {} else {
         Add-Type @"
-	using System.Net;
+    using System.Net;
     using System.Security.Cryptography.X509Certificates;
     public class TrustAllCertificatePolicy : ICertificatePolicy {
         public TrustAllCertificatePolicy() {}
-		public bool CheckValidationResult(
+        public bool CheckValidationResult(
             ServicePoint sPoint, X509Certificate certificate,
             WebRequest wRequest, int certificateProblem) {
             return true;
         }
-	}
+    }
 "@
         [System.Net.ServicePointManager]::CertificatePolicy = New-Object TrustAllCertificatePolicy
     }
 }
+
+
+#######################################################################################################################
+########################################################  FUNCTIONS  ##################################################
+
+
+Function Get-vCenterServer {
```

Comment:
**Question**: Would this wrapper function be better as `Connect-vCenterServer`?

Reply:
Decided on not to change this.